testdata/scripts/typeparams.txt

```diff
@@ -1,25 +1,53 @@
-[!go1.18] skip 'type parameters require Go 1.18 or later'
+[go1.18] gofumpt foo.go
+[!go1.18] ! gofumpt foo.go
 
-gofumpt foo.go
-cmp stdout foo.go.golden
+[go1.18] cmp stdout foo.go.golden
+[!go1.18] stderr 'foo.go:.*expected'
 
 -- go.mod --
 module test
 
 go 1.18
 -- foo.go --
-package main
+package p
 
-func Foo[   A, B any](x A, y B) {}
```

Comment:
assuming it was intentional to rm the space before `A`

Reply:
I used a dummy space elsewhere in the file :) don't think it really matters